README.md

```diff
@@ -34,16 +34,20 @@ The widget is a ready-to-use drop-in bundle with a CSS and customizable template
   <img src="/images/WidgetAnimation.gif" alt="JavaScript Widget for the PingFederate Authentication API">
 </p>
 
-## PingFederate Configuration
+# PingFederate Configuration
 
 PingFederate acts as the server interacting with the widget via APIs to authenticate the user.
 
 To configure PingFederate for the widget:
-  1. Enable the authentication API: Authentication > Authentication API Applications > Enable Authentication API.
-  1. Add an application: Add Authentication Application > Name: TestApp, URL:  `https://localhost:8443` > Save.
-  1. Create a Password Credential Validator, HTML Form Adapter, and an IdP Connection that uses the adapter for SSO. Alternatively, use OAuth Playground flows to authenticate using the HTML Form Adapter.
-  1. Create a policy: Authentication > Policies > Create (or update an existing policy) that uses the HTML Form Adapter > Authentication Application > Select the previously created app from the dropdown.
-  1. Start SSO flow where you will get redirected to the app.
+  1. First enable the authentication API: Authentication > Authentication API Applications > Enable Authentication API.
+  2. Then, add an application by clicking the "Add Authentication Application" button and entering the appropriate values. For example: **Name:** TestApp, **URL:** `https://localhost:8443`
+  3. Click "Save".
+  
+  **Caution:** setting your Authentication Application as the "Default Authentication Application" will make it the default authentication for all of your existing connections. This is the easiest way to configure your connections, but it
+  is not very precise. For more precision, configure the desired adapter to us your Authentication API Application.
```

Comment:
Since authn applications are chosen in at the policy level, maybe it would make more sense to say "configure the desired authentication policies to use your Authentication API Application"?

Reply:
Was this comment resolved on purpose @jameslamothe-ping? 

@philipcoldrey-ping thoughts on this one?

Reply:
Good catch @mketchum. Let me make that update.